credits.txt

```diff
@@ -157,63 +164,69 @@ contributed to Endless Sky:
   DeBlister
   DingusShingleton
   Disiuze
+  DJF113
+  DownsB
   dplepage
+  dragonmaus
```

Comment:
You had one more commit other than just that https://github.com/endless-sky/endless-sky/commits/master?author=dragonmaus (and have since added another).